<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddActionFrom</class>
 <widget class="QWidget" name="AddActionFrom">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetAction">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="Say">
    <attribute name="title">
     <string>Say</string>
    </attribute>
    <widget class="QLabel" name="lbNameSay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name action:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtNameSay">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtSay">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>291</width>
       <height>131</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbSay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Say:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ActionBasic">
    <attribute name="title">
     <string>Joint Action</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetActionBasic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Plock">
    <attribute name="title">
     <string>Partical Lock</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxALL">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ALL</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Right hand</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHL">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Left hand</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxF">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Foot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxH_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>joint_9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxH_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>joint_10</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxH_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>joint_11</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxH_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>joint_6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxH_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>joint_7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxH_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>joint_8</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Zip">
    <attribute name="title">
     <string>Action</string>
    </attribute>
    <widget class="QListWidget" name="lwdZip">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSAVE">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>AddAction</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPoseButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>181</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>C:/Users/Dat/OneDrive/Desktop/Nhap_mon_tuong_tac_nguoi_may-main/pose.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="NewPoseButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>181</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>C:/Users/Dat/OneDrive/Desktop/Nhap_mon_tuong_tac_nguoi_may-main/pose.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnJointAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>EditAction</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
